src/app/models/evt-models.ts

```diff
@@ -105,3 +105,27 @@ export interface NamedEntityRefData extends GenericElementData {
     entityId: string;
     entityType: NamedEntityType;
 }
+
+export interface WitnessesData {
+    witnesses: Map<Witness>;
+    witnessGroups: WitnessGroups;
+}
+
+export interface Witness {
+    id: string;
+    name: Description;
+    attributes: AttributesData;
+    content: Description;
+    group: string;
+}
+
+export interface WitnessGroups {
+    groups: Map<WitnessGroup>;
+}
+export interface WitnessGroup {
+    id: string;
+    name: string;
```

Comment:
Are you sure that this is a simple string and not a `GenericElementData[]` as for the `Witness`?

Reply:
I saw that it can contain also a `<head>` or `<desc>`, thus it may be more complex than a string

Reply:
Nevermind, it is probably enough to have simply string for now